docs/pages/database-access/guides/aws-cross-account.mdx

```diff
@@ -0,0 +1,196 @@
+---
+title: AWS Cross-Account Database Access
+description: How to connect AWS databases in external AWS accounts to Teleport.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="13.0"
+>
+  AWS cross-account database access is available starting from Teleport `13.0`.
+</Details>
+
+You can deploy the Teleport Database Service with AWS IAM credentials in one
+AWS account and use AWS IAM roles to grant Teleport access to databases in
+external AWS accounts.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- A Teleport Database Service to connect your AWS database(s) to Teleport.
+- AWS account(s) with permissions to modify and attach IAM policies.
+- AWS VPC network configured to allow the Teleport Database Service to connect
+  to your database(s).
+
+## Teleport configuration
+
+The Teleport Database Service must be configured to assume an external AWS IAM
+role and, optionally, pass an external ID when it assumes that role.
+For information about when you should use an external ID, see:
+[purpose of an AWS external ID](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html#external-id-purpose).
```

Comment:
I think understanding this paragraph is the key to understanding this entire guide, so it would make sense to expand it a bit, especially for AWS users who may be dealing with external IDs for the first time (I'm guessing that's the audience of this guide, given the link to the AWS docs?).

- How does the Database Service assume an external AWS IAM role? 
- What is an external ID and how does the Database Service use it?
- What does "pass an external ID" mean? What is the Database Service passing the external ID to?

Reply:
Expanded on this here: 1559d2b09239080858cd90a593914abbb71e53c2